### ISSUE TITLE
How to convert a list into a set?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#17]
How to convert a list into a set?

+ User can convert list into set by using `set()` function.

### Example:
```python
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sat’,’sun’,’tue’]
listAsSet = set(weekdays)
print(listAsSet)
```
### output
```bash
set([‘wed’, ‘sun’, ‘thu’, ‘tue’, ‘mon’, ‘fri’, ‘sat’])
```

**Comment by AshishNamdev** [comment]
Included in [Q11](../blob/master/questions/Q1-50.md#Q11)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
